pkg/commands/compute/build.go

```diff
@@ -151,11 +169,24 @@ func (c *BuildCommand) Exec(in io.Reader, out io.Writer) (err error) {
 		}
 	}
 
-	if c.Manifest.File.Scripts.Build != "" {
-		lang = "custom"
+	// NOTE: A third-party could share a project with a build command for a
+	// language that wouldn't normally require one (e.g. Rust), and do evil
+	// things. So we should notify the user and confirm they would like to
+	// continue with the build.
+	if !c.Flags.AcceptCustomBuild && language.Name == "other" && toolchain == "custom" {
+		label := "This project has a custom build script defined in the fastly.toml manifest. Are you sure you want to continue with the build step? [y/N] "
+		cont, err := text.Input(out, label, in)
+		if err != nil {
+			return fmt.Errorf("error reading input %w", err)
+		}
+		contl := strings.ToLower(cont)
+		if contl != "y" && contl != "yes" {
+			text.Info(out, "Stopping the build process.")
+			return nil
+		}
 	}
```

Comment:
Would it be too much to always display `This project has a custom build script defined in the fastly.toml manifest` in verbose mode?